functional/tdx/run.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+#
+# Copyright (c) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -x
```

Comment:
do we need this?

Reply:
it's not essential, but I'd like to keep it to make debugging easier

Reply:
fwiw, you could add the following as the CI jobs all seem to set `DEBUG=true`:

```bash
[ -n "${DEBUG:-}" ] && set -o xtrace
```